components/data-components/change-language-button.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @next/next/no-img-element */
```

Comment:
it's better if you create an .eslintrc file with a rule to mute that warning instead of putting it in every file.